docs/data-sources/ip_ranges.md

```
@@ -0,0 +1,56 @@
+
+### Read-Only
+
+- `all_prefixes` (List of String) Filtered IPv4 and IPv6 CIDR prefixes.
+- `create_date` (String) Creation timestamp returned by the UptimeRobot IP ranges endpoint.
+- `ipv4_prefixes` (List of String) Filtered IPv4 CIDR prefixes.
+- `ipv6_prefixes` (List of String) Filtered IPv6 CIDR prefixes.
+- `prefixes` (Attributes List) Filtered monitoring IP prefixes with normalized CIDR and source fields. (see [below for nested schema](#nestedatt--prefixes))
+- `sync_token` (String) Sync token returned by the UptimeRobot IP ranges endpoint.
+
+<a id="nestedatt--prefixes"></a>
+### Nested Schema for `prefixes`
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)